scripts/lint/lint.sh

```diff
@@ -47,7 +48,7 @@ function run_tests() {
   ALL_FILES=$1
   PYTHON_FILES=$2
 
-  run "ruff" ruff check $PYTHON_FILES --quiet
```

Comment:
@maxime-desroches is there a reason this was done? we always want to run the same thing a user would do, and this was masking an error I hit while running "ruff check ."

Reply:
Yes, the goal was to only check files that are tracked by git and ignore new files that you might have created that are unrelated or that you don't want to commit